src/shared/components/login/additionalInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Section from 'shared/components/section/section';
+import Form from 'shared/components/form/form';
+import FormZipCode from 'shared/components/form/formZipCode/formZipCode';
+import FormPassword from 'shared/components/form/formPassword/formPassword';
+import FormButton from 'shared/components/form/formButton/formButton';
+import styles from 'scenes/home/informationForm/informationForm.css';
+import _ from 'lodash';
+import { patchBackend } from 'shared/utils/apiHelper';
+
+class AdditionalInfo extends React.Component {
+  state = {
+    zip: '',
+    password: ''
+  }
+  onZipChange = (value, valid) => {
```

Comment:
Perhaps add a single space in between each one of these assignments (throughout the PR), for increased readability.  

For example:

```javascript
state = {
  zip: '',
  password: ''
}

onZipChange = (value, valid) => {
  this.setState({ zip: value, zipValid: valid });
}

onPasswordChange = (value, valid) => {
  this.setState({ password: value, passwordValid: valid });
}
```